### DISCUSSION TITLE
Is this package compatible with Laravel 11  ?

### DISCUSSION THREAD

**Discussion by resi-pawan** [ctwillie/expo-server-sdk-php#43] [Q&A] [answered]
Package hasn't received update since Dec 2023.

**Comment by ctwillie** [comment, marked as answer]
@resi-pawan Hey. Laravel 11 requires "php": "^8.2", and this packages requires "php": ">=7.3".. So yes, it is compatible. I just tested on a fresh v11 project with no issues.

This package has very little dependencies and has 99% code coverage, so very little maintenance has been required so far.
